LaTeX/TeX.sublime-syntax

```diff
@@ -108,6 +154,55 @@ contexts:
       scope: constant.character.space.tex
     - include: escaped-character
 
+  tex-constants:
+    - match: (\\){{constants}}{{endcs}}
+      scope: constant.language.tex
+      captures:
+        1: punctuation.definition.backslash.tex
+      push:
+        - tex-constant-value
+        - tex-constant-assignment
+    - match: (\\){{macroconst}}{{endcs}}
+      scope: constant.language.tex
+      captures:
+        1: punctuation.definition.backslash.tex
+
+  tex-constant-assignment:
+    - match: =
+      scope: keyword.operator.assignment.tex
+      pop: 1
+    - include: else-pop
+    - include: paragraph-pop
+
+  tex-constant-value:
+    - match: ((-?)\d+(\.)\d*)({{dimunits}}?)
+      scope: meta.number.float.decimal.tex
+      captures:
+        1: constant.numeric.value.tex
+        2: keyword.operator.arithmetic.tex
+        3: punctuation.separator.decimal.tex
+        4: constant.numeric.suffix.tex
+    - match: ((-?)\d+)({{dimunits}}?)
+      scope: meta.number.integer.decimal.tex
+      captures:
+        1: constant.numeric.value.tex
+        2: keyword.operator.arithmetic.tex
+        3: constant.numeric.suffix.tex
+    - match: (`)(\\)?({{charbycode}}|.)
+      scope: meta.number.integer.tex
+      captures:
+        1: keyword.operator.tex
+        2: punctuation.definition.backslash.tex
+        3: constant.character.tex
+    - match: '{{dimunits}}'
+      scope: constant.numeric.suffix.tex
+
+    - match: plus|minus
```

Comment:
Are word breaks required to ensure not to highlight `plus` within `someplustext` or is something like `plus5` valid?